spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/PoolControllerHelperService.java

```diff
@@ -768,6 +774,7 @@ public JSONObject getPoolableElementInfo(HttpSession session, JSONObject json) {
                   + dilution.getLibrary().getAlias() + "(" + dilution.getLibrary().getName() + ")</a><br/>");
               info.append("<b>Sample:</b> <a href='/miso/sample/" + dilution.getLibrary().getSample().getId() + "'>"
                   + dilution.getLibrary().getSample().getAlias() + "(" + dilution.getLibrary().getSample().getName() + ")</a><br/>");
+              info.append("<b>Percentage:</b> " + Math.round(dilution.getConcentration() * 100 / total) + "%<br/>");
```

Comment:
I don't think that the lab treats percent pool content like this -- whenever I've seen them discuss it, it's a "Please make this pool 25% Dilution A, 25% Dilution B, and 50% Dilution C." In this case, it has to be user-entered on a per-pool basis, not calculated.